test/cosign_test.py

```diff
@@ -3,48 +3,233 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
+import hashlib
+import json
+
+import cryptography.hazmat.primitives.asymmetric.rsa as rsa
+import cryptography.hazmat.primitives.serialization as crypto_serialiation
+import dacite
 import pytest
+import requests.exceptions
 
 import cosign
+import model.signing_server
+import oci.model
+
+
+class OciClientMock:
```

Comment:
how about using `unittest.mock` intead of a custom mock? this would have the advantage that it has builtin method-calling-capturing-capabilities (see https://docs.python.org/3/library/unittest.mock.html#quick-guide), which can be quite useful in unittests

Reply:
By factoring-out a `sign_manifest` function, there is no need for an `oci_client`-mock for the unit test anymore.